tech/languages/csharp/dotnet-installation.md

```diff
@@ -0,0 +1,23 @@
+---
+title: C# - .NET Core
+subsection: csharp
+section: tech-languages
+order: 2
+version: 7.8.4
+description: "C# programming language, .NET Core and Mono open source frameworks."
+---
+
+# .NET SDK installation
+
+.NET Core is still work in progress, and below download locations contain packages that are not entirely _free._ Please be patient, we should have reasonable packages within a few months.
+
+Choose based on your Fedora and SDK version:
+
+* [Microsoft's 2.0 preview1](https://www.microsoft.com/net/core/preview#linuxfedora) (F25 `.csproj`)
+* [Microsoft's 1.X packages](https://www.microsoft.com/net/download/linux) (F23 & 24 `.json`)
+* [Fedora .NET SIG packages](https://copr.fedorainfracloud.org/coprs/nmilosev/dotnet-sig) (F25+ `.json`)
```

Comment:
There are references to F23, 24, 25, but on FDP we intend to support only latest stable Fedora (currently F26).